vault.sh

```diff
@@ -105,13 +105,13 @@ HELP
 
     # clear vars being set
     aws_reset
-
+    TTL=${TTL:-2h}
```

Comment:
This is a great idea!
What if we make the TTL more unique to vault?
What do you think about renaming to VAULT_TTL?

Reply:
VAULT_TTL is more clear than TTL.